gitlab/resource_gitlab_project.go

```diff
@@ -208,6 +208,22 @@ var resourceGitLabProjectSchema = map[string]*schema.Schema{
 		Type:     schema.TypeBool,
 		Optional: true,
 	},
+	"template_name": {
```

Comment:
For attributes that cannot be updated after resource creation, like these, we use the `ForceNew: true` field ([source](https://github.com/hashicorp/terraform-plugin-sdk/blob/35d7fcb6c861606651dee1584261138dd71fb61b/helper/schema/schema.go#L140)). Could you add that to these new attributes?